frontend/openchat-client/src/stores/throttling.ts

```diff
@@ -0,0 +1,62 @@
+import { writable } from "svelte/store";
+
+const localStorageKey = "openchat_most_recent_sent_message_times";
+
+const countdown = writable<number>(0);
+export const throttleCountdown = {
```

Comment:
Could just store the timestamp and pass that to ThrottleCountdown and use $now500?

Reply:
Yes I think that would be better. This all evolved before I did the UI part, so I think you're right it can be simpler.